web/gds-user-ui/src/components/Section/JoinUs.tsx

```diff
@@ -95,22 +95,41 @@ const JoinUsSection: React.FC = () => {
                 </Box>
               ))}
             </SimpleGrid>
-            <Box alignItems="center" textAlign="center">
+            <Stack 
+              justifyContent={"center"}
+              alignItems={"center"} 
+              direction={['column', 'row']} 
+              spacing={4}
+              >
               <RouterLink to={'/guide'}>
                 <Button
                   bg="#FF7A59"
                   color="white"
                   borderColor="white"
                   py={6}
-                  maxWidth={'190px'}
-                  width="100%"
+                  width="190px"
```

Comment:
Good question! When the `View Member List` button was added, the `Join Today` button's width decreased and was much smaller. This app uses chakra ui components and after attempting to use the `Flex` and `Grid` components, I noticed that the buttons still didn't have the same width, so setting the width on `Join Today` made it match the `View Member List` button. I increased the screen size and noticed the `Join Today` button's width didn't increase, so that's why `maxWidth` was removed.

Reply:
thanks for the detailed explanation, @daniellemaxwell!